src/styles/globals.css

```
@@ -174,6 +174,7 @@
 @import './themes/catppuccin.css';
 @import './themes/t3chat.css';
 @import './themes/claude.css';
+@import './decorum.css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)